### PR TITLE
Add quick sample workflow

### PR DESCRIPTION
**Pull request by ldecicco-USGS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,57 @@
 ScienceBase R Tools
 ===
+
 [![Build Status](https://travis-ci.org/USGS-R/sbtools.svg?branch=master)](https://travis-ci.org/USGS-R/sbtools) [![codecov.io](https://codecov.io/github/USGS-R/sbtools/coverage.svg?branch=master)](https://codecov.io/github/USGS-R/sbtools?branch=master)
 Tools for interfacing R with ScienceBase data services.
 
+
+##Package Installation
+To install the dataRetrieval package, you must be using R 3.0 or greater and run the following command:
```
Comment (aappling-usgs) [inline README.md]: To install the `sbtools` package, you mean?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,57 @@
...
+
+```r
+library(sbtools)
+authenticate_sb()
+temp.directory <- tempdir()
+item_file_download('548b2b31e4b03f64633662a4', names='gdp_new.txt',
+		destinations=file.path(temp.directory, 'gdp_new.txt'))
+read.table(file.path(temp.directory, 'gdp_new.txt'), sep = ":", header = FALSE)
+   V1     V2
+1 RT   blkah
```
Comment (aappling-usgs) [inline README.md]: what a nice table =)

and in all seriousness, great idea to add a sample workflow to the README

File: README.md (hunk 1 above)
Comment (ldecicco-USGS) [inline README.md]: can you possibly guess where I pasted this from?

File: README.md (hunk 2 above)
Comment (ldecicco-USGS) [inline README.md]: I want to send an email out to the USGS-R users today. If you can make a more relevant workflow to get people started, that would be great.

Comment (aappling-usgs) [inline README.md]: Luke put some good ones in the manuscript. I'll dig up 1-2.

**Pull request merged**